sydent/http/servlets/store_invite_servlet.py

```diff
@@ -126,7 +126,7 @@ def render_POST(self, request: Request) -> JsonDict:
             substitutions.setdefault(k, "")
 
         # MSC3288
-        substitutions["room_type"] = substitutions.pop(
+        substitutions["room_type"] = substitutions.get(
```

Comment:
I'm not sure if we should change this. My hope was the templates didn't need to know about the unstable prefix.

Reply:
I agree that templates shouldn't be using these, however I think that we'd need to document that not everything that gets sent to store-invite is accessible... What about if we put an "unstable name" warning next to the two popped variables? (Or don't doccument them until they're made stable, but still make them available)

Reply:
I've marked them as unstable.